pkg/network/tracer/connection/perf_batching.go

```diff
@@ -176,9 +176,12 @@ func (p *perfBatchManager) cleanupExpiredState(now time.Time) {
 
 func newConnBatchManager(mgr *manager.Manager) (*perfBatchManager, error) {
 	connCloseEventMap, _, err := mgr.GetMap(probes.ConnCloseEventMap)
-	if err != nil || connCloseEventMap == nil {
+	if err != nil {
 		return nil, fmt.Errorf("unable to get map %s: %s", probes.ConnCloseEventMap, err)
 	}
+	if connCloseEventMap == nil {
+		return nil, fmt.Errorf("unable to get map %s: cannot find a map with that name", probes.ConnCloseEventMap)
+	}
 
 	connCloseMap, err := ebpfutil.GetMap[uint32, netebpf.Batch](mgr, probes.ConnCloseBatchMap)
```

Comment:
Same for this here? Need to check if `connCloseMap == nil`?

Reply:
Not in that case, that [GetMap function](https://github.com/DataDog/datadog-agent/blob/5c2fc1534511c8c117c5fff46e339d4dc6e46ddc/pkg/ebpf/util/generic_map.go#L116) does the check for the nil map. I realize it's a bit confusing/inconsistent, I'll check with @brycekahle to see if we can merge this in the ebpf-manager repo later on and make it a bit more uniform.